src/supportedK8sVersions.json

```diff
@@ -1 +1 @@
-{ "supportedK8sVersions": ["1.25", "1.26", "1.27"] }
+{ "supportedK8sVersions": ["1.26", "1.27", "1.28"] }
```

Comment:
We can add 1.28 without deprecating 1.25 yet.

Reply:
Updated `supportedK8sVersions` and `package.json` without deprecating 1.25 for v2.9.